### ISSUE TITLE
Publish unminified code

### ISSUE THREAD

**Issue by poma** [keitetran/BroadlinkIRTools#6]
It is really hard to follow changes in your minified code and not possible to contribute fixes to your repo. Also hard to verify that your code doesn't send private data to your server.

**Comment by keitetran** [comment]
:( i dont need that infomation. Please check it on network of browser. Press F12 and try connect to other server. Or y can create a vitual URL of hass and create token. Remove token after use this tool. 
That is safe.

**Comment by keitetran** [comment]
if i publish code, this url can not working https://keitetran.github.io/BroadlinkIRTools/ we need a server for HTML and JS working.

**Comment by poma** [comment]
You can publish it in addition to minified code, so that other users can clone your repo, read code, and help you fix bugs. Your html can still reference minified version.

**Comment by keitetran** [comment]
OK i will publish that after i finish my job
